rest/taskana-rest-spring/src/main/java/pro/taskana/rest/RestConfiguration.java

```diff
@@ -88,4 +98,10 @@ public Jackson2ObjectMapperBuilder jacksonBuilder(HandlerInstantiator handlerIns
     public HandlerInstantiator handlerInstantiator(ApplicationContext context) {
         return new SpringHandlerInstantiator(context.getAutowireCapableBeanFactory());
     }
+
+    @Override
+    protected void configureMessageConverters(List<HttpMessageConverter<?>> converters) {
```

Comment:
I am not sure why are you using this. Where can I have a look to this when it is activated?